concepts/50 React Components/70 TypeScript Support/05 Scopes of Types/15 Common Types.md

```diff
@@ -0,0 +1,36 @@
+**Modular Projects**
+
+Common types are usually in the `ComponentTypes` namespace. If a common type is missing, file an issue in our [Support Center](https://supportcenter.devexpress.com/ticket/create).
+
+When using a type for multiple components, it can be unclear from which module to import:
+
+    import DataGrid, { DataGridTypes } from 'devextreme-react/data-grid';
+    import Popup, { PopupTypes } from 'devextreme-react/pop-up';
+
+    const toolbarItemsLocation: DataGridTypes.ToolbarItemLocation = 'center';
+    // or
+    const toolbarItemsLocation: PopupTypes.ToolbarItemLocation = 'center';
+
+In such cases, import the type from a common submodule:
+
+    import { ToolbarItemLocation } from 'devextreme-react/common';
+    const toolbarItemsLocation: ToolbarItemLocation;
+
+Alternatively, define a union:
```

Comment:
Add here a link to [05 Using Unions Instead of Base Classes.md](https://github.com/DevExpress/devextreme-documentation/pull/6454/files#diff-7cc287644072e3d40ad5f86168fc780981c8005ed8d747fe8d473378a85617ff) (kinda "See XXX for more details")